### ISSUE TITLE
I wonder if I can insert a formula by using the $$...$$

### ISSUE THREAD

**Issue by howie-wu** [marktext/marktext#3334]
### Question description

I wonder if I can insert a formula by using the $$...$$?
If so how does that work?

**Comment by jgoossen851** [comment]
The README indicates that [KaTeX expressions](https://katex.org/docs/supported.html) are supported for inputting formulas. You can insert formulas by enclosing them with a single `$`. I don't think the double `$$` syntax is supported.
E.g.: `$\int{}f(x)\ dx$` = $\int{}f(x)\ dx$

**Comment by YalandHong** [comment]
`$$` is used for formula block.

![image](https://user-images.githubusercontent.com/52960068/179879048-96f6489e-b744-4bec-8a10-351ad5d65585.png)

**Comment by tyongguang** [comment]
Support $$ is very important for me. It's really heartbreaking that this issue is still open.

**Comment by Jocs** [comment]
Yes, `$$...$$` math blocks are fully supported in MarkText. Type `$$` on a new line, press Enter, and a math block editor will appear where you can enter LaTeX. Inline math uses single `$...$`. Closing as answered.
